assets/src/edit-story/components/dropzone/utils/disableDefaults.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export default function disableDefaults(evt) {
+  evt.preventDefault();
+  evt.stopPropagation();
+}
```

Comment:
This seems a little superfluous. I'd rather be specific about whether to both prevent default and stop propagation inline.